po/POTFILES

```diff
@@ -2,6 +2,7 @@ share/lutris/ui/about-dialog.ui
 share/lutris/ui/dialog-lutris-login.ui
 share/lutris/ui/dialog-pga-sources.ui
 share/lutris/ui/dialog-uninstall-game.ui
+share/lutris/ui/lutris-window.ui
```

Comment:
Could be that this will conflict with #2961.

Reply:
But in my previous PR, @YihaoPeng asked me to include this to make the missing strings available, so I assume it'll be okay then.

Reply:
> Could be that this will conflict with #2961.

Two identical changes will not cause conflicts, git will handle these intelligently.